In-house patch to remove unnecessary dependencies from Keystone's
requirements files. The specific reasons are as follows:

oslo.serialization	Not applicable

paste			Not applicable

posix_ipc		Not applicable

--- keystone-2014.2.2/keystone.egg-info/requires.txt.~1~	2015-02-05 06:56:18.000000000 -0800
+++ keystone-2014.2.2/keystone.egg-info/requires.txt	2015-02-24 22:46:53.872644685 -0800
@@ -4,7 +4,6 @@ eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
 netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes>=1.12.3,!=2.0
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
@@ -17,11 +16,9 @@ oslo.config>=1.4.0  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.i18n>=1.0.0  # Apache-2.0
-oslo.serialization>=1.0.0               # Apache-2.0
 oslo.utils>=1.0.0                       # Apache-2.0
 Babel>=1.3
 oauthlib>=0.6
 dogpile.cache>=0.5.3
 jsonschema>=2.0.0,<3.0.0
-pycadf>=0.6.0
-posix_ipc
+pycadf>=0.6.0
\ No newline at end of file
--- keystone-2014.2.2/requirements.txt.~1~	2015-02-05 06:53:51.000000000 -0800
+++ keystone-2014.2.2/requirements.txt	2015-02-24 22:46:14.319449987 -0800
@@ -9,7 +9,6 @@ eventlet>=0.15.1,<0.16.0
 greenlet>=0.3.2
 netaddr>=0.7.12
 PasteDeploy>=1.5.0
-Paste
 Routes>=1.12.3,!=2.0
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
@@ -22,12 +21,9 @@ oslo.config>=1.4.0  # Apache-2.0
 oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
 oslo.db>=1.0.0,<1.1  # Apache-2.0
 oslo.i18n>=1.0.0  # Apache-2.0
-# only for Keystone tests running against keystoneclient master
-oslo.serialization>=1.0.0               # Apache-2.0
 oslo.utils>=1.0.0                       # Apache-2.0
 Babel>=1.3
 oauthlib>=0.6
 dogpile.cache>=0.5.3
 jsonschema>=2.0.0,<3.0.0
 pycadf>=0.6.0
-posix_ipc
